src/Akka.TestKit.NUnit/TestKit.cs

```diff
@@ -68,26 +65,14 @@ public TestKit(string config)
         public new static Config FullDebugConfig { get { return TestKitBase.FullDebugConfig; } }
 
         protected static NUnitAssertions Assertions { get { return _assertions; } }
-
-        /// <summary>
-        /// This method is called before each test run, it initializes the test including
-        /// creating and setting up the ActorSystem.
-        /// </summary>
-        [SetUp]
-        public void InitializeActorSystemOnSetUp()

```

Comment:
Setup no longer needed now that this can be done from `ActorSystem` constructor.